stripe/resource.py

```diff
@@ -55,6 +55,16 @@ def convert_to_stripe_object(resp, api_key, account):
         return resp
 
 
+def convert_array_to_dict(arr):
+    if isinstance(arr, list):
+        d = {}
+        for i, value in enumerate(arr):
+            d["{0}".format(i)] = value
```

Comment:
Yeah, I avoided that because I'm not sure how it works when we use `2to3`. I'm sure it works